src/main/java/sg/ncl/DataController.java

```diff
@@ -485,8 +615,26 @@ public String getStatistics(Model model,
             downloadStats.put(statInfoObject.getInt("dataId"), statInfoObject.getLong("count"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "dataId" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)